### ISSUE TITLE
POSTGRES: invalid defaultValue

### ISSUE THREAD

**Issue by zucatti** [sequelize/sequelize-auto#551]
model generated :
    insertedAt: {
      type: DataTypes.DATE,
      allowNull: false,
      defaultValue: "timezone(utc",
      field: 'inserted_at'
    }
with a field like   inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL

Thanks again for all the work you've done with sequelize-auto

**Comment by steveschmitt** [comment]
Yes, we have some problem with the default value functions.  Thanks for another example.
